features/format.feature

```diff
@@ -410,10 +410,12 @@ Feature: Custom formats
         """
         And the content of file "2020-08-29_entry-the-first.md" in the cache should be
         """
+        ---
         title: Entry the first.
         date: 2020-08-29 11:11
         starred: False
         tags: tagone, ipsum, tagtwo
+        ...
```

Comment:
I think the example in #1065 meant the dots here as a placeholder for content later in the file (not actual dots).

Reply:
Oh, hey, these dot actually part of the yaml spec. My mistake!

https://yaml.org/spec/1.2/spec.html#id2760395